### DISCUSSION TITLE
Man page task

### DISCUSSION THREAD

**Discussion by Vlek** [wolandark/bash-dungeon#13] [Ideas]
I know that this could get a bit overwhelming for a new user, but asking them to look through the man page for something and use that information for a task would probably be good?

This one most likely needs a hint as well.

**Comment by Vlek** [comment]
By the way, sorry for bombarding your repo with stuff. I just was super excited and loved the project.

**Comment by wolandark** [comment]
> could get a bit overwhelming for a new user

I agree with this, 
in room1, there is a task that requires the user to grep the output of --help. I think that's a great lesson to teach a new user. 

Although learning to use the man or info pages is more valuable in the long run, I can't imagine anyone managing to properly find something in the man page without at least a limited understanding of regex searches. 

At this point, the issue of the manpager comes to mind. While I think it's best to use Vim as a manpager, many might not agree with me. 
Although even less, as a manpager supports some degree of regex.

For example: finding the `-d` flag in `man tr` with `/^\s*-d` could be a part of the task, but the HINT file wont be brief.


> By the way, sorry for bombarding your repo with stuff

That's perfectly fine and I appreciate your passion and help.
